mlflow/models/cli.py

```diff
@@ -77,14 +78,56 @@ def predict(model_uri, input_path, output_path, content_type, json_format, no_co
                                                                       json_format=json_format)
 
 
+@commands.command("build-docker")
+@cli_args.MODEL_URI
+@click.option("--name", "-n", default="mlflow-pyfunc-servable",
+              help="Name to use for built image")
+def build_docker(model_uri, name):
```

Comment:
you can add the `@experimental` decorator here (right above the `def`)

Reply:
Yep good call, unfortunately the annotation makes the help string for build-docker look less informative (for some reason everything after the initial ".." is cut) when running `mlflow models --help`:


```
(mlflow_python3) ➜  mlflow git:(docker-model-build) ✗ mlflow models --help
Usage: mlflow models [OPTIONS] COMMAND [ARGS]...

  Deploy MLflow models locally.

  To deploy a model associated with a run on a tracking server, set the
  MLFLOW_TRACKING_URI environment variable to the URL of the desired server.

Options:
  --help  Show this message and exit.

Commands:
  build-docker  ..
  predict       Generate predictions in json format using a saved MLflow...
  serve         Serve a model saved with MLflow by launching a webserver on...
```

That being said I can add the annotation if the downside of not having it is that we'd forget that this API is experimental (or I could try to add an argument like `no_doc` to the annotation to avoid modifying the docstring in this special case)

Reply:
Ah that's ok. I don't think we'll forget this is experimental if the docstring itself says so. I'd just format it to be the same as other experimental notes.